user-guide/Troubleshooting/Known_issues/KI_DataMinerMessageBroker.md

```diff
@@ -0,0 +1,21 @@
+---
+uid: KI_DataMinerMessageBroker
+---
+
+# Max Payload exceptions occur when using MessageBroker with chunking
+
+## Affected versions
+
+From DataMiner 10.3.5 [CU0]/10.3.8 onwards.
```

Comment:
I'm a bit confused about the affected versions. These are both Feature Release versions. Is it "DataMiner 10.3.5 [CU0] and 10.3.8" instead? Is a Main Release version affected? @LaurensVergote

Reply:
I think I messed up the numbers. It's 10.3.0(CU5)/10.3.8

Reply:
Fixed in commit [a443791](https://github.com/SkylineCommunications/dataminer-docs/pull/1995/commits/a44379164a16a60a54b2f279463af80b310b4545)